TestCase/Managers/InputManager.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Runtime.Intrinsics.X86;
+
+namespace TestCase.Managers
+{
+    public class InputManager
+    {
+        public int GetInput()
+        {
+            Console.Write("\nEnter number between 1 and 3: ");
+
+            if (int.TryParse(Console.ReadLine(), out int num))
+            {
+                if (num >= 1 && num <= 3)
+                {
+                    return num;
+                }
+                else
+                {
+                    Console.WriteLine("\nyou must give a number between 1 and 3");
+                    return GetInput();
+                }
+            }
+            Console.WriteLine("\nPlease enter a number between 1 and 3");
+            return GetInput();
+        }
+
+        public bool ResetChoiceInput()
+        {
+            Console.WriteLine();
+            Console.WriteLine("\nwould you like to play again. ");
+
+            Console.Write("\nY for yes or N for no. ");
+
+            string resetChoice = Console.ReadLine();
+
+            if (resetChoice == "Y")
```

Comment:
```suggestion
            if (resetChoice.Compare("y", StringComparison.OrdinalIgnoreCase))
```
Same with 'n'. Itll compare ignoring case removing the extra checks.

Reply:
this one as well was a bit iffy

Reply:
Sorry, I got confused with the method. It should have been `string.Equals(resetChoice, "y", StringComparison.OrdinalIgnoreCase)`. Tested in .net fiddle (online C# playground to run code) and it works as expected:
![image](https://github.com/RaeesHussain9/TicTacToe/assets/72302841/1be972c7-976f-48b4-a7fd-b55f5b918771)